ext/xmlwriter/php_xmlwriter.stub.php

```diff
@@ -6,85 +6,85 @@ function xmlwriter_open_uri(string $uri): XMLWriter|false {}
 
 function xmlwriter_open_memory(): XMLWriter|false {}
 
-function xmlwriter_set_indent(XMLWriter $xmlwriter, bool $indent): bool {}
+function xmlwriter_set_indent(XMLWriter $writer, bool $enable): bool {}
 
-function xmlwriter_set_indent_string(XMLWriter $xmlwriter, string $indentString): bool {}
+function xmlwriter_set_indent_string(XMLWriter $writer, string $indentation): bool {}
 
-function xmlwriter_start_comment(XMLWriter $xmlwriter): bool {}
+function xmlwriter_start_comment(XMLWriter $writer): bool {}
 
-function xmlwriter_end_comment(XMLWriter $xmlwriter): bool {}
+function xmlwriter_end_comment(XMLWriter $writer): bool {}
 
-function xmlwriter_start_attribute(XMLWriter $xmlwriter, string $name): bool {}
+function xmlwriter_start_attribute(XMLWriter $writer, string $name): bool {}
 
-function xmlwriter_end_attribute(XMLWriter $xmlwriter): bool {}
+function xmlwriter_end_attribute(XMLWriter $writer): bool {}
 
-function xmlwriter_write_attribute(XMLWriter $xmlwriter, string $name, string $value): bool {}
+function xmlwriter_write_attribute(XMLWriter $writer, string $name, string $content): bool {}
 
-function xmlwriter_start_attribute_ns(XMLWriter $xmlwriter, ?string $prefix, string $name, ?string $uri): bool {}
+function xmlwriter_start_attribute_ns(XMLWriter $writer, ?string $prefix, string $name, ?string $uri): bool {}
 
-function xmlwriter_write_attribute_ns(XMLWriter $xmlwriter, ?string $prefix, string $name, ?string $uri, string $content): bool {}
+function xmlwriter_write_attribute_ns(XMLWriter $writer, ?string $prefix, string $name, ?string $uri, string $content): bool {}
 
-function xmlwriter_start_element(XMLWriter $xmlwriter, string $name): bool {}
+function xmlwriter_start_element(XMLWriter $writer, string $name): bool {}
 
-function xmlwriter_end_element(XMLWriter $xmlwriter): bool {}
+function xmlwriter_end_element(XMLWriter $writer): bool {}
 
-function xmlwriter_full_end_element(XMLWriter $xmlwriter): bool {}
+function xmlwriter_full_end_element(XMLWriter $writer): bool {}
 
-function xmlwriter_start_element_ns(XMLWriter $xmlwriter, ?string $prefix, string $name, ?string $uri): bool {}
+function xmlwriter_start_element_ns(XMLWriter $writer, ?string $prefix, string $name, ?string $uri): bool {}
 
-function xmlwriter_write_element(XMLWriter $xmlwriter, string $name, ?string $content = null): bool {}
+function xmlwriter_write_element(XMLWriter $writer, string $name, ?string $content = null): bool {}
```

Comment:
Aha, I see why content was used for the attribute cases! While it is's a good term for describing what an element contains, it's not the best for the value of an attribute.

Reply:
I understand your preference for `$value` for attributes. I would be OK with using `$value` for attributes while keeping `$content` elsewhere. Are you OK with that as well @nikic?

Reply:
Yeah, I'm okay with making that distinction.

Reply:
Ok, I just pushed these last few changes.